### ISSUE TITLE
Simplify initialization and setup

### ISSUE THREAD

**Issue by dyladan** [open-telemetry/opentelemetry-specification#500]
tl;dr setup is too complex in most cases. Should SIGs abstract it away themselves and set sane defaults or wait for spec?

Right now in the current state of the JS repo, the end-user is required to initialize multiple parts of the SDK: the tracer provider, the propagator, the context manager, and the meter provider. Currently this looks like this:

```typescript
import api from "@opentelemetry/api";
import { NodeTracerProvider } from "@opentelemetry/node";
import { MeterProvider } from "@opentelemetry/metrics";
import { AsyncHooksScopeManager } from "@opentelemetry/scope-async-hooks";
import { HttpTraceContext } from "@opentelemetry/base";

api.context.initGlobalContextManager(new AsyncHooksScopeManager());
api.trace.initGlobalTracerProvider(new NodeTracerProvider());
api.metrics.initGlobalMeterProvider(new MeterProvider());
api.propagation.initGlobalPropagator(new HttpTraceContext());

// User code below this line
```

Obviously this is not really ideal, and we would like to simplify this setup. Before OTEP 66, we had this (the node tracer provider assumed a particular default scope manager and propagator):

```typescript
import api from "@opentelemetry/api";
import { NodeTracerProvider } from "@opentelemetry/node";
import { MeterProvider } from "@opentelemetry/metrics";

api.trace.initGlobalTracerProvider(new NodeTracerProvider());
api.metrics.initGlobalMeterProvider(new MeterProvider());
```

Ideally, we would go back to the simpler setup we had before. Ultimately, my question is this: how much freedom do SIG maintainers have to simplify setup and assume "sane" defaults, and how much of this should we be waiting for the spec to decide?

**Comment by dyladan** [comment]
We talked about this in the spec SIG meeting and the general agreement is that it's ok for individual SDK SIGs to do whatever they feel is appropriate here as long as it doesn't violate the spec or API.
